pkg/stack/unwind/unwind_table.go

```diff
@@ -188,10 +188,18 @@ func (ptb *UnwindTableBuilder) PrintTable(writer io.Writer, path string) error {
 		return err
 	}
 
+	// The frame package can raise in case of malformed unwind data.
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Fprintf(writer, "recovered a panic in PrintTable %v\n", r)
```

Comment:
I think we need to use the logger in the `ptb`.

Reply:
oops!